### ISSUE TITLE
[feature request] Add cross to image frame

### ISSUE THREAD

**Issue by Boorj** [CarbonDesigns/carbon-ui#352]
Usually, images on prototypes are drawn with diagonal cross. Like this

![image](https://user-images.githubusercontent.com/2145313/31758742-e4d2ca02-b4d8-11e7-8044-4befdac0f09a.png)

Also it will be useful to make image differ.

![image](https://user-images.githubusercontent.com/2145313/31758855-5442db16-b4d9-11e7-8170-bac283feffe2.png)

On this picture it's intuitively seen that right rectangle is an image.

**Comment by Boorj** [comment]
Is it hard to do or not? @defr0zen @alex-carbonium

**Comment by defr0zen** [comment]
No, but low priority. There should probably be a setting for this since not always you'd want these crosses.

**Comment by Boorj** [comment]
i disagree, it could be everywhere, without setting to disable that. It's not a big problem to have that, i suppose.
